### PR TITLE
add Optimism Goerli chain

### PR DESCRIPTION
**Pull request by klyap**
## Description

Adding Optimism Goerli as a supported chain

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/wagmi/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address: kerls.eth

### CODE REVIEW

File: packages/core/src/constants/blockExplorers.ts [hunk 1]
```diff
@@ -46,6 +47,10 @@ export const etherscanBlockExplorers: Record<EtherscanChains, BlockExplorer> = {
     name: 'Etherscan',
     url: 'https://kovan-optimistic.etherscan.io',
   },
+  optimismGoerli: {
+    name: 'Etherscan',
+    url: 'https://l2-explorer.surge.sh/',
```
Comment (tmm) [inline packages/core/src/constants/blockExplorers.ts:52]: This isn't an official Etherscan block explorer so let's remove this from here. Can inline this URL on the `blockExplorers.default` field for now.

**Review by tmm — CHANGES_REQUESTED** [review]
Small change, then good

File: packages/core/src/constants/blockExplorers.ts (hunk 1 above)
Comment (klyap) [reply packages/core/src/constants/blockExplorers.ts:52]: Should I just use 'https://goerli-optimistic.etherscan.io' in the `url` field here instead? Or pass an empty string?

Also, what do you mean by inlining in `blockExplorers.default`? Like inline comment?

Comment (tmm) [reply packages/core/src/constants/blockExplorers.ts]: Etherscan doesn't have a block explorer out yet for Goerli Optimism so no need to add to the Etherscan block explorer object.

**Review by tmm — APPROVED** [review]

**Pull request merged**